### PR TITLE
rename brain connectome and anonymize for publication

### PR DESCRIPTION
**Pull request by johmathe**
rename brain connectome and anonymize for publication

### CODE REVIEW

File: nips2018/geomstats/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Computations and statistics on manifolds with geometric structures.
 
-<img src="https://raw.githubusercontent.com/ninamiolane/geomstats/master/examples/imgs/gradient_descent.gif" width=384 height=384><img src="https://raw.githubusercontent.com/ninamiolane/geomstats/master/examples/imgs/h2_grid.png" width=256 height=256>
+<img src="https://raw.githubusercontent.com/xxxxxx/geomstats/master/examples/imgs/gradient_descent.gif" width=384 height=384><img src="https://raw.githubusercontent.com/xxxxxx/geomstats/master/examples/imgs/h2_grid.png" width=256 height=256>
```
Comment (ninamiolane) [inline nips2018/geomstats/README.md:6]: doesn't impact the image's plots?

File: nips2018/geomstats/LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
```
Comment (ninamiolane) [inline nips2018/geomstats/LICENSE.md:1]: The abbreviations in the file names of brain_connectome are not very clear: what is "comp_ind" in comp_ind_fmri.csv ?

rs_fmri_fnc_mapping.csv : rs stands for resting state, but what is fnc?

Maybe change this, unless this is the naming used in the (open-source) data (link in the paper).

File: nips2018/geomstats/examples/gradient_descent_s2.py [hunk 3]
```diff
@@ -92,8 +92,8 @@ def main(output_file='out.mp4', max_iter=128):
     previous_x = initial_point
     geodesics = []
     n_steps = 20
-    # TODO(johmathe): auto differentiation
-    # TODO(johmathe): gpu implementation
+    # TODO(xxx): auto differentiation
+    # TODO(xxx): gpu implementation
```
Comment (ninamiolane) [inline nips2018/geomstats/examples/gradient_descent_s2.py:96]: Haha no xxx and xxy?

File: nips2018/geomstats/examples/tangent_pca_so3.py [hunk 4]
```diff
@@ -13,7 +13,7 @@
 METRIC = SO3_GROUP.bi_invariant_metric
 
 N_SAMPLES = 20
-# TODO(nina): find real data set
+# TODO(xxx): find real data set
```
Comment (ninamiolane) [inline nips2018/geomstats/examples/tangent_pca_so3.py:16]: let's delete that one x)

File: nips2018/geomstats/geomstats/special_euclidean_group.py [hunk 5]
```diff
@@ -409,7 +409,7 @@ def random_uniform(self, n_samples=1):
         """
         random_rot_vec = self.rotations.random_uniform(n_samples)
         random_translation = self.translations.random_uniform(n_samples)
-        # TODO(nina): remove this line after full parallelization
+        # TODO(xxx): remove this line after full parallelization
```
Comment (ninamiolane) [inline nips2018/geomstats/geomstats/special_euclidean_group.py:412]: this line should be removed now.

File: nips2018/geomstats/tests/test_examples.py [hunk 6]
```diff
@@ -39,11 +39,11 @@ def test_plot_geodesics_se3(self):
     def test_plot_geodesics_so3(self):
         plot_geodesics_so3.main()
 
-    # TODO(nina): this test fails
+    # TODO(xxx): this test fails
     # def test_plot_grid_h2(self):
     #    plot_grid_h2.main()
 
-    # TODO(johmathe): this test fails
+    # TODO(xxx): this test fails
```
Comment (ninamiolane) [inline nips2018/geomstats/tests/test_examples.py:46]: We should debug these....

File: nips2018/geomstats/examples/gradient_descent_s2.py (hunk 3 above)
Comment (johmathe) [reply nips2018/geomstats/examples/gradient_descent_s2.py:96]: nah I realized this actually gives a hint on how many contributors we have :)

File: nips2018/geomstats/LICENSE.md (hunk 2 above)
Comment (johmathe) [reply nips2018/geomstats/LICENSE.md:1]: yeah I asked the same thing to claire - still don't have an answer :)

File: nips2018/geomstats/tests/test_special_euclidean_group.py [hunk 7]
```diff
@@ -1476,7 +1476,7 @@ def test_dist_vectorization(self):
                             'with metric {}'.format(metric_type))
 
     def test_group_exponential_barycenter(self):
-        # TODO(nina): this test fails, the barycenter is not accurate.
+        # TODO(xxx): this test fails, the barycenter is not accurate.
```
Comment (ninamiolane) [inline nips2018/geomstats/tests/test_special_euclidean_group.py:1479]: Does this still fail?

File: nips2018/geomstats/tests/test_visualization.py [hunk 8]
```diff
@@ -15,7 +15,7 @@
 S2 = Hypersphere(dimension=2)
 
 
-# TODO(nina): add tests for examples
+# TODO(xxx): add tests for examples
```
Comment (ninamiolane) [inline nips2018/geomstats/tests/test_visualization.py:18]: This is done now. We can remove.

**Review by ninamiolane — APPROVED** [review]

**Pull request merged**